bindings/python/README.md

```diff
@@ -52,6 +46,12 @@ Activate venv:
 source venv/bin/activate
 ````
 
+Install `maturin`:
+
+```shell
+pip install maturin[patchelf]
```

Comment:
`patchelf` is only required for `maturin build` to produce portable wheels for distribution, `maturin develop` doesn't need it.

Reply:
Just to eliminate

```
⚠️ Warning: Failed to set rpath for /home/psiace/work/opendal/target/debug/libopendal.so: Failed to execute 'patchelf', did you install it? Hint: Try `pip install maturin[patchelf]` (or just `pip install patchelf`)
```

:smiley: